proto/dydxprotocol/clob/tx.proto

```diff
@@ -26,6 +27,8 @@ service Msg {
   rpc PlaceOrder(MsgPlaceOrder) returns (MsgPlaceOrderResponse);
   // CancelOrder allows accounts to cancel existing orders on the orderbook.
   rpc CancelOrder(MsgCancelOrder) returns (MsgCancelOrderResponse);
+  // BatchCancel allows accounts to cancel a batch of orders on the orderbook.
+  rpc BatchCancel(MsgBatchCancel) returns (MsgBatchCancelResponse);
```

Comment:
you might need to add the corresponding methods to `msgServer` so that it implements the defined interface